docs/README.md

```diff
@@ -134,7 +134,7 @@ The [Search Box component](components/search-box.md) renders a text input.
 
 The text input value is bound to the query of the current search.
 
-Every time the query changes, the search Store will contact Algolia to get the new results for the new query.
```

Comment:
many places we used capitalization without any true reason/meaning. Now it's only used when we speak about specific components or at the beginning of sentences.